### PR TITLE
v2.1.0 Release

### PR DESCRIPTION
**Pull request by JanBobolz**

### CODE REVIEW

**Comment by rheitjoh** [comment]
This needs to be a v2.1.0 version bump due to the new debug group API.

**Comment by JanBobolz** [comment]
RIght. Good point!

File: README.md [hunk 1]
```diff
@@ -68,15 +67,15 @@ To add the newest Math version as a dependency, add this to your project's POM:
 <dependency>
     <groupId>org.cryptimeleon</groupId>
     <artifactId>math</artifactId>
-    <version>1.0.0</version>
+    <version>2.0.0</version>
```
Comment (rheitjoh) [inline README.md:70]: Needs to be a 2.1.0 now.

File: README.md [hunk 2]
```diff
@@ -68,15 +67,15 @@ To add the newest Math version as a dependency, add this to your project's POM:
...
 </dependency>
 ```
 
 ### Installation With Gradle
 
 Math is published via Maven Central.
 Therefore, you need to add `mavenCentral()` to the `repositories` section of your project's `build.gradle` file.
-Then, add `implementation group: 'org.cryptimeleon', name: 'math', version: '1.0.0'` to the `dependencies` section of your `build.gradle` file.
+Then, add `implementation group: 'org.cryptimeleon', name: 'math', version: '2.0.0'` to the `dependencies` section of your `build.gradle` file.
```
Comment (rheitjoh) [inline README.md:78]: 2.1.0

File: README.md [hunk 3]
```diff
@@ -86,7 +85,7 @@ repositories {
 }
 
 dependencies {
-    implementation group: 'org.cryptimeleon', name: 'math', version: '1.0.0'
+    implementation group: 'org.cryptimeleon', name: 'math', version: '2.0.0'
```
Comment (rheitjoh) [inline README.md:88]: 2.1.0

**Review by rheitjoh — APPROVED** [review]

**Pull request merged**